.github/script/STEP

```diff
@@ -1 +1 @@
-0
+2
```

Comment:
```suggestion
2
```instead of two we can also try it for 4 players as a 3D game where its 9x9 cube
so that they can choose if want to play via 2 or 4 players